src/mod.ts

```diff
@@ -1,2 +1,2 @@
 export * from "./types.ts";
-export { I18n } from "./i18n.ts";
+export * from "./i18n.ts";
```

Comment:
This not only exports `hears`, but also `CTX`. Is this intended?

Reply:
That wasn't intended either. I'll re-work on exports there.